Base/usr/share/man/man6/GameOfLife.md

```diff
@@ -0,0 +1,21 @@
+## Name
+
+GameOfLife
+
+## Synopsis
+
+```**sh
+$ GameOfLife
+```
+
+## Description
+
+GameOfLife is a implementation of John Conway's Game of Life.
+
+The game is a cellular automata where each cell is either dead (grey) or alive (yellow) and will change state in the next tick if any of the following rules are fulfilled:
+
+* An alive cell will die by underpopulation if it has fewer than two neighbours.
+* An alive cell will die by overpopulation if it has more than three neighbours.
+* A dead cell will come alive by reproduction if it has exactly three neighbours.
+
+otherwise it will keep its old state.
```

Comment:
```suggestion
GameOfLife is an implementation of John Conway's Game of Life.

The game is a cellular automaton where each cell is either dead (grey) or alive (yellow) and will change state in the next tick if any of the following rules are fulfilled:

* An alive cell will die by underpopulation if it has fewer than two neighbors.
* An alive cell will die by overpopulation if it has more than three neighbors.
* A dead cell will come alive by reproduction if it has exactly three neighbors.

Otherwise, it will keep its old state.
```